README.rst

```diff
@@ -1,48 +1,151 @@
 
-   npm install
-   npm start # The server will run on port 1999
+.. list-table:: Environment Variables
+   :widths: 30 50 20
+   :header-rows: 1
 
-Once the dev server is up visit http://localhost:1999/login.
+   * - Name
+     - Description / Usage
+     - Example
 
-Configuration and Deployment
-----------------------------
+   * - ``LOGIN_ISSUE_SUPPORT_LINK``
+     - The fully-qualified URL to the login issue support page in the target environment.
+     - ``https://support.example.com``
 
-This MFE is configured via node environment variables supplied at build time. See the .env file for the list of required environment variables. Example build syntax with a single environment variable:
+   * - ``ACTIVATION_EMAIL_SUPPORT_LINK``
+     - The fully-qualified URL to the activation email support page in the target environment.
+     - ``https://support.example.com``
 
-.. code:: bash
+   * - ``PASSWORD_RESET_SUPPORT_LINK``
+     - The fully-qualified URL to the password reset support page in the target environment.
+     - ``https://support.example.com``
 
-   NODE_ENV=development ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload' npm run build
+   * - ``WELCOME_PAGE_SUPPORT_LINK``
+     - The fully-qualified URL to the welcome support page in the target environment.
+     - ``https://support.example.com``
 
+   * - ``TOS_AND_HONOR_CODE``
+     - The fully-qualified URL to the Honor code page in the target environment.
+     - ``https://example.com/honor``
+
+   * - ``TOS_LINK``
+     - The fully-qualified URL to the Terms of service page in the target environment.
+     - ``https://example.com/tos``
+
+   * - ``PRIVACY_POLICY``
+     - The fully-qualified URL to the Privacy policy page in the target environment.
+     - ``https://example.com/privacy``
+
+   * - ``INFO_EMAIL``
+     - The valid email address for information query regarding the target environment.
+     - ``info@example.com``
+
+   * - ``ENABLE_DYNAMIC_REGISTRATION_FIELDS``
+     - Enables support for dynamic registration fields that helps in adding more fields in register form for getting more details about users.
+     - ``true`` | ``''`` (empty strings are falsy)
+
+   * - ``DISABLE_ENTERPRISE_LOGIN``
+     - Enables support for disabling enterprise login.
+     - ``true`` | ``''`` (empty strings are falsy)
+
+edX-specific Environment Variables
+**********************************
+
+Furthermore, there are several edX-specific environment variables that enable integrations with closed-source services private to the edX organization, and might be unsupported in Open edX.
+
+.. list-table:: edX-specific Environment Variables
+   :widths: 30 50 20
+   :header-rows: 1
+
+   * - Name
+     - Description / Usage
+     - Example
+
+   * - ``ENABLE_PROGRESSIVE_PROFILING``
+     - Enables support for progressive profiling that helps in redirecting to welcome page and getting more details about users.
```

Comment:
```suggestion
     - Enables support for progressive profiling. If enabled, users are redirected to a second page where data for optional registration fields can be collected. 
```

Reply:
Now that I think about it, I think this should be moved to the above table. The progressive profiling page is needed for the optional fields and should be available to openedX users as well.